packages/react-events/src/dom/Scroll.js

```diff
@@ -165,13 +230,19 @@ const ScrollResponder: ReactDOMEventResponder = {
         state.pointerType = pointerType;
         break;
       }
+      case 'wheel': {
+        state.pointerType = 'mouse';
```

Comment:
Did you mention that `trackPad` fires both `scroll` and `wheel` events?

Reply:
I mentioned it in the summary of this PR. Did you mean for me to add a comment somewhere too?

Reply:
Is the pointerType going to be correct here if trackPad is the source of the wheel event?

Reply:
There's no way to know if it's a trackpad or not. Mouse events can trigger wheel events too.

Reply:
OK sounds like we'll just not have trackpad as a pointer type for scroll